packages/jest-config/src/validConfig.js

```diff
@@ -55,9 +55,13 @@ module.exports = ({
   modulePathIgnorePatterns: ['<rootDir>/build/'],
   modulePaths: ['/shared/vendor/modules'],
   name: 'string',
+  noDefaultReporters: false,
```

Comment:
i actually think it's a really good idea.
so it will be
```js
"reporters": [
  "default",
  ["custom-jest-reporter-package", {configValue: true}]
}
```

```
"additionalReporters": [
  ["custom-jest-reporters-package", {configValue: true}]
]
```
?